scijava/scijava-ops-api/src/main/java/org/scijava/ops/api/OpEnvironment.java

```diff
@@ -746,9 +746,9 @@ default OpBuilder.Arity16 arity16(final String opName, final Hints hints) {
 	/**
 	 * Creates some {@link OpInfo}s from {@code o}
 	 * @param o the {@link Object} to create {@link OpInfo}s from
-	 * @return a {@link Collection} of {@link OpInfo}s
+	 * @return a {@link Set} of {@link OpInfo}s
 	 */
-	Collection<OpInfo> infosFrom(Object o);
+	Set<OpInfo> infosFrom(Object o);
```

Comment:
Why `Set` rather than `List`? If we use `List` then we have an ordering, and we can return them ordered by priority, which is useful in some scenarios. Right now, `infosFrom` uses `Collectors.toSet`, you `descriptions` was changed to use a set also, but `opsFromObject` still uses `Collectors.toList`. I'd like to be very intentional here in each case about whether ordering would be a good thing or a bad thing, and type accordingly.